packages/backend/src/services/ProjectService/ProjectService.ts

```diff
@@ -1166,9 +1175,14 @@ export class ProjectService {
                         'warehouse.type',
                         warehouseClient.credentials.type,
                     );
-                    const { rows } = await warehouseClient.runQuery(
-                        query,
-                        queryTags,
+                    const { rows } = await wrapSentryTransaction(
+                        'runWarehouseQuery',
+                        {
+                            query,
```

Comment:
query is the sqlQuery right ? is this more useful than metricQuery ?

Reply:
Thanks for spotting this, I want to see: 
- if the sqlQuery could be monitored used as part of tracing so we can check something in Sentry (I know it's already available in Grafana, but trying to maximise the usage with Sentry)
- Added stringified `metricQuery` (don't know?)
- Added `context` to see where this query is being run